CHANGELOG.md

```diff
@@ -5,11 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 
+## Gaprindshavili-2 released 2018-03-06
+
+### Fixed
+- Fix refresh for stack without parameters [(#410)](https://github.com/ManageIQ/manageiq-providers-amazon/pull/410)
+
 ## Unreleased as of Sprint 80 ending 2018-02-26
 
 ### Fixed
 - Raise Exception if AMI not visible [(#411)](https://github.com/ManageIQ/manageiq-providers-amazon/pull/411)
-- Fix refresh for stack without parameters [(#410)](https://github.com/ManageIQ/manageiq-providers-amazon/pull/410)
 - Return created topic [(#409)](https://github.com/ManageIQ/manageiq-providers-amazon/pull/409)
```

Comment:
@JPrause - Im curious why this was moved?

Reply:
@bronaghs in this case, the fix in PR410, went from unreleased (not in an upstream official release) to being backported for inclusion in an official upstream release. Only select PRs are backported based on PM review and/or blocker status.

Reply:
Ah i see, thanks for clarifying.